server/README.md

```diff
@@ -22,11 +22,24 @@ To be able to open Keycloak on localhost map port 8080 locally
 
 ## Creating admin account
 
-By default there is no admin user created so you won't be able to login to the admin console. To create an admin account
-you need to use environment variables to pass in an initial username and password. This is done by running:
+To be able to login to the admin console you need to have admin user.  
+To create an admin account with your login and password you need to use environment variables to pass in an 
+initial username and password. This is done by running:
 
     docker run -e KEYCLOAK_USER=<USERNAME> -e KEYCLOAK_PASSWORD=<PASSWORD> jboss/keycloak
 
+When you run the container without `KEYCLOAK_USER` and `KEYCLOAK_PASSWORD` user and password are generated 
+automatically, you can find it with:
```

Comment:
Start a new sentence, ```You can find the actual username / password of the admin account with:```. Yet, the code below should be modified to display the username / password credentials combination for both possible cases, how Keycloak can be started (IOW the ones specified via ```KEYCLOAK_USER``` / ```KEYCLOAK_PASSWORD``` combination, or the autogenerated ones).